lib/routes/project.js

```diff
@@ -164,6 +164,9 @@ module.exports = function (modulus) {
     this.line('    -m, --include-modules   Flag that indicates whether or not to include the node_modules folder in the bundle.'.input);
     this.line('    -r, --registry          Specify a registry to install modules from.'.input);
     this.line('    -w, --with-tests        Specifies that tests should be run before deploying this project.'.input);
+    this.line('        --nodeVersion       Specified version of node to use when running application.'.input);
```

Comment:
Those are typos. I just corrected them to be `--node-version` and `--npm-version`